backend/src/main/java/com/shipping/backend/services/ShippingRequestSenderImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.shipping.backend.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ShippingRequestSenderImpl implements  ShippingRequestSender{
+
+    private final static Logger log = LoggerFactory.getLogger(ShippingRequestSenderImpl.class);
+
+    @Value("${config-queue.name}")
+    private String QUEUE_NAME;
+
+    @Value("${config-queue.exchange}")
```

Comment:
`@Value` is a fast and nice way to binding your properties values to a variable.
What happens if you are using a lot of `@Values` of the same properties, by example `config-queue.exchange` in more than 15 classes and at the end, you need to change the name of the property on each class?
Please read this article and make the change please
https://tuhrig.de/why-using-springs-value-annotation-is-bad/

Reply:
As we discuss las saturday i will add a calsa to get all the values from the code.